optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/result/ProblemBenchmarkResult.java

```diff
@@ -380,7 +380,7 @@ public void registerScale(long registeringEntityCount, long registeringVariableC
             long registeringProblemScale) {
         if (entityCount == null) {
             entityCount = registeringEntityCount;
-        } else if (entityCount.longValue() != registeringEntityCount) {
+        } else if (entityCount != registeringEntityCount) {
```

Comment:
I would just like to remind you that these are automated code changes - I sincerely doubt that IDEA would be recommending these changes if they had even the slightest potential of actually being harmful. (And, of course, no tests failed.)